src/Repository/WishlistRepository.php

```diff
@@ -71,6 +72,20 @@ public function findAllByShopUserAndToken(int $shopUser, string $token): ?array
             ;
     }
 
+    public function countByShopUserAndToken(int $shopUser, string $token): int
+    {
+        return (int) $this
+            ->createQueryBuilder('o')
+            ->select('COUNT(o.id)')
+            ->where('o.shopUser = :shopUser')
+            ->orWhere('o.token = :token')
```

Comment:
You're right, but I have add this method as a mirror of `findAllByShopUserAndToken` which does an `or` even if the name says `and`.

I'm not in favor of a different query between the two methods, so we either need to rename both method name or change both method query, but in the end it will introduce a change in the logic or a BC break. 😕